lib/grape-swagger/entity/parser.rb

```diff
@@ -62,8 +62,9 @@ def parse_grape_entity_params(params)
               }
             end
 
-            if entity_options[:values] && entity_options[:values].is_a?(Array)
-              memo[entity_name][:enum] = entity_options[:values]
+            if entity_options[:documentation] && entity_options[:documentation][:values]
```

Comment:
Why not to use `entity_options[:documentation][:values] || entity_options[:values]`?